### PR TITLE
Implement first version of `Edusign::Client`

### PR DESCRIPTION
**Pull request by grmnlrt**
Implemented first version of `Edusign::Client` based on [Kitt's `EdusignClient`](https://github.com/lewagon/kitt/blob/master/app/services/edusign_client.rb).

### CODE REVIEW

File: lib/edusign/client.rb [hunk 1]
```diff
@@ -1,5 +1,278 @@
 module Edusign
   class Client
-    # TODO(grmnlrt): put Kitt code here.
+    include HTTParty
```
Comment (ssaunier) [inline lib/edusign/client.rb:5]: Needs to be declarend as dependency in the gemspec

**Review by ssaunier — CHANGES_REQUESTED** [review]

File: lib/edusign/client.rb (hunk 1 above)
Comment (grmnlrt) [reply lib/edusign/client.rb:5]: The GHA is still failing, do I need to run `bundle install` after adding the dependency? 🤔

Comment (ssaunier) [reply lib/edusign/client.rb:5]: I think so!

Comment (ssaunier) [reply lib/edusign/client.rb:5]: Also maybe

```
bundle lock --add-platform x86_64-linux
```

cf GHA logs

Comment (grmnlrt) [reply lib/edusign/client.rb:5]: Actually I needed to `require "httparty"` as well 😅 
I forgot I wasn't on rails

**Pull request merged**